experiment/kind-e2e.sh

```diff
@@ -71,9 +71,15 @@ build() {
     kind build node-image --type=bazel --kube-root="${PWD}"
 
     # make sure we have e2e requirements
-    #make all WHAT="cmd/kubectl test/e2e/e2e.test vendor/github.com/onsi/ginkgo/ginkgo"
+    #make all WHAT="cmd/kubectl test/e2e/e2e.test vendor/github.com/onsi/ginkgo/ginkgo cluster/images/conformance/go-runner"
     bazel build //cmd/kubectl //test/e2e:e2e.test //vendor/github.com/onsi/ginkgo/ginkgo
```

Comment:
It would in the follow up. If I put it there and it merges before that is in k/k then this will fail for everyone.

If I ignore pipefail on that line then it ignores legit failures on the other sections (and still requires a followup to remove the pipefail allowance)

Reply:
it's a lot more expensive to invoke bazel twice in CI (AFAIK we don't keep a background server around, and it needs to run the analysis phase each time)

maybe you could probe for the directory existing instead?